hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieSnapshotExporter.java

```diff
@@ -121,6 +126,20 @@ public static class Config implements Serializable {
 
     @Parameter(names = {"--parallelism", "-pl"}, description = "Parallelism for file listing")
     public int parallelism = 0;
+
+    @Parameter(names = {"--transformer-class"}, description = "A subclass of org.apache.hudi.utilities.transform.Transformer"
+            + ". Allows transforming raw source Dataset to a target Dataset (conforming to target schema) before "
+            + "writing. Default: Not set. Available transformers: org.apache.hudi.utilities.transform.SqlQueryBasedTransformer, "
+            + "org.apache.hudi.utilities.transform.SqlFileBasedTransformer, org.apache.hudi.utilities.transform.FlatteningTransformer.")
+    public String transformerClassName = null;
+
+    @Parameter(names = {"--transformer-sql"}, description = "sql-query template be used to transform the source before"
```

Comment:
Looks like these two options are relient on the `--transformer-class`, are they required or optional, can we log some prompt msgs if they are missing?

Reply:
`--transformer-sql` is required for `SqlQueryBasedTransformer` transformer class, `--transformer-sql-file` is required for `SqlFileBasedTransformer`, in other  cases they are ignored. Added validation of these options to `main` method with logging appropriate messages.